### DISCUSSION TITLE
Hand Tracking 2.0 Support

### DISCUSSION THREAD

**Discussion by RanmaKei** [alvr-org/ALVR#1054] [Q&A] [answered]
Is Hand Tracking 2.0 for Quest 2 automatically supported or does this require any changes?

**Comment by zmerp** [comment, marked as answer]
It has been added recently, nightly releases should have support.

**Reply by ColdIce1605** [reply]
To clarify: yes it requires changes and those changes are recently in nightly(as of date in comment) and as of my comment  has been in v18.2.2 which has been out for awhile now.
